tests/unit/test_file_handle_cache.py

```diff
@@ -0,0 +1,62 @@
+"""
+Tests the file handle cache
+"""
```

Comment:
Sorry, should have spotted this earlier. We're missing some standard  `__future__` imports here. These are essential for future Python 3 compatibility.